test/parser/expression_defs_test.cpp

```diff
@@ -0,0 +1,160 @@
+#include "parser/expression_defs.h"
+#include <string>
+
+#include "gtest/gtest.h"
+
+namespace terrier::parser {
+
+// NOLINTNEXTLINE
+TEST(ExpressionDefsTests, ExpressionTypeToStringTest) {
```

Comment:
This function is gross. Why not just make a map of input / outputs?

Reply:
a map would also require us to hard code all its input-output pairs